test/helpers/testWrapper.ts

```diff
@@ -0,0 +1,25 @@
+/**
+ * Copyright 2023 Fluence Labs Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { TEST_DEFAULT_SENDER_ACCOUNT } from "./localNetAccounts.js";
+
+export const maybeConcurrentTest = (...args: Parameters<typeof test>): void => {
+  console.log(
+    `Running "${args[0]}" with sender account: ${TEST_DEFAULT_SENDER_ACCOUNT.address}`,
+  );
+
+  test.concurrent(...args);
+};
```

Comment:
I'm standing against this wrapper. Adding just `console.log` above each test is kinda repetitive, but much simpler and cleaner. If you want to add some logs before each test in the suite, i suggest to use `beforeEach` hook for that specific suit.
Also, why would you need to do `test.concurrent` ? Isn't `it.concurrent` working?

Reply:
You are right, it should be deleted.
There was another logic in that function. I left it for debug purpose while setting up distribution of private keys in parallel test launch.

`test` and `it` are the same. So it doesn't matter what do you use.